engines/aws/app/models/aws/dns_record.rb

```diff
@@ -27,6 +27,7 @@ def aws_destroy
       @cli.delete_dns_record(
         hosted_zone_id, fqdn.value, hostname, 'CNAME'
       )
+      throw(:abort) unless Rails.configuration.lasso_run.present?
```

Comment:
Explain, please.

Reply:
All the models have that `throw`, the `aws_destroy` is a `before_destroy: aws_destroy` hook in [aws_resource](https://github.com/SUSE-Enceladus/suse-rancher-setup/blob/main/engines/aws/app/models/aws/aws_resource.rb#L5), meaning if one does not interrupt the call, it will destroy the resource which should only happen when destroying the resources and not when generating the commands script